modules/platforms/dotnet/Apache.Ignite.Core.Tests/Services/ServicesTest.cs

```diff
@@ -1328,6 +1341,15 @@ public interface ITestIgniteService : IService, ITestIgniteServiceBase
 
             /** */
             object ErrMethod(object arg);
+
+            /** */
+            int testOverload(int count, Employee[] emps);
```

Comment:
Please follow .NET naming conventions - public member names should start with a capital letter.

Reply:
Fixed. Thanks.